packages/ui-karma-config/lib/index.js

```diff
@@ -179,14 +180,13 @@ module.exports = function makeConfig({ coverageDirectory, coverageThreshold }) {
           ]
         },
         mode: 'development',
-        devtool: NO_DEBUG ? 'none' : 'inline-source-map',
-
-        //TODO: this is probably a hack and webpack 5 will remove these features
-        //investigate how to do this properly
-        node: { fs: 'empty', module: 'empty' },
-        resolve: {
-          extensions: baseWebpackConfig.resolve.extensions
-        },
+        devtool: NO_DEBUG ? false : 'inline-source-map',
+        plugins: [
+          // fix Buffer/process is not defined errors:
+          new webpack.ProvidePlugin({ process: 'process/browser' }),
+          new webpack.ProvidePlugin({ Buffer: ['buffer', 'Buffer'] })
```

Comment:
Can't we simply call this only one time?
```js
new webpack.ProvidePlugin({
    process: 'process/browser',
    Buffer: ['buffer', 'Buffer']
  }),
```

Reply:
yes, done